java/pom.xml

```diff
@@ -78,6 +81,7 @@
         <mockito.version>3.3.3</mockito.version>
         <testcontainers.version>1.16.2</testcontainers.version>
         <wiremock.version>2.32.0</wiremock.version>
+        <assertj.version>3.23.1</assertj.version>
```

Comment:
Can we update the NOTICES file with this new dependency?

Reply:
Done